tests/test_strings.py

```diff
@@ -350,6 +350,20 @@ def test_lines(case, expected):
 
 
 @parametrize('case,expected', [
+    ('fooBar', 'foo bar'),
+    ('--foo-Bar--', 'foo bar'),
+    ('*Foo*B_a*R', 'foo b a r'),
+    ('*Foo10_B*Ar', 'foo10 b ar'),
+    ('/?*Foo10/;"B*Ar', 'foo10 b ar'),
+    ('/?F@O__o10?>.B?>";Ar', 'f o o10 b ar'),
+    ('FoO Bar', 'fo o bar'),
+    ('F\n\soO Bar', 'f so o bar')
+])
```

Comment:
I would ran all these examples through Lodash because a few of them don't match.